Lib/threading.py

```diff
@@ -1027,6 +1031,13 @@ def _bootstrap_inner(self):
             self._set_ident()
             if _HAVE_THREAD_NATIVE_ID:
                 self._set_native_id()
+            if _set_name is not None and self._name:
+                try:
+                    _set_name(self._name)
+                except UnicodeEncodeError:
+                    # cannot set the name if the name cannot be encoded
+                    # to the filesystem encoding
+                    pass
```

Comment:
With `"replace"`, this shouldn't happen.
But, maybe we should ignore any `Exception`, or at least any `OSError`. Who knows what limits there are on non-tiered platforms; not being able to set the OS name shouldn't prevent a thread from starting.

Reply:
I replaced it with `except OSError: pass` for now.